jsonschema-validation.xml

```diff
@@ -1005,7 +1014,14 @@
                 <t>
                     If the instance is a string, this property indicates the media type
                     of the contents of the string.  If "contentEncoding" is present,
-                    this property describes the decoded string.
+                    this property describes the decoded string.  If the "type" keyword is
+                    absent, this keyword MAY be interpreted as describing an unencoded binary
+                    resource.  The exact meaning and behavior of this untyped usage is
+                    application-defined.
+                    <cref>
+                        For an example of application-defined untyped usage,
+                        see the forthcoming OpenAPI Specification v3.1
```

Comment:
Should a link to this be added?

Reply:
There isn't a link to provide yet. A cref kinda doesn't matter too much.